apps/src/templates/sectionProgressV2/MoreDetailsDialog.jsx

```diff
@@ -12,11 +12,12 @@ import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
 import {ITEM_TYPE} from './ItemType';
 import ProgressIcon from './ProgressIcon';
 import styles from './progress-key-popup.module.scss';
+import iconStyles from './progress-table-legend.module.scss';
 
 export default function MoreDetailsDialog({hasValidation, onClose}) {
-  const renderItem = (itemType, itemTitle, itemDetails) => (
+  const renderItem = (itemType, itemTitle, itemDetails, className) => (
```

Comment:
For most of the icons you can omit this last parameter, and pass in `color.neutral_dark` for the choice level icon.
```suggestion
  const renderItem = (itemType, itemTitle, itemDetails, colorOverride) => (
```